api/registrations/views.py

```diff
@@ -86,7 +89,7 @@ def get_node(self, check_object_permissions=True):
         return node
 
 
-class RegistrationList(JSONAPIBaseView, generics.ListAPIView, bulk_views.BulkUpdateJSONAPIView, NodesFilterMixin):
+class RegistrationList(JSONAPIBaseView, generics.ListCreateAPIView, bulk_views.BulkUpdateJSONAPIView, NodesFilterMixin, DraftMixin):
```

Comment:
Old workflow to create a registration is a POST to nodes/<node_id>/registrations/.  New workflow is a POST to /registrations/.  Still supporting old workflow.  Old workflow and new workflow need to be clearly documented.